packages/x-charts/src/models/axis.ts

```diff
@@ -106,6 +106,11 @@ export interface ChartsAxisProps extends TickParams {
    * @default 'auto'
    */
   tickLabelInterval?: 'auto' | ((value: any, index: number) => boolean);
+  /**
+   * The minimum space between ticks when using an ordinal scale. It defines the minimum distance in pixels between two ticks.
+   * @default 0
```

Comment:
```suggestion
```

Reply:
Why adding a default instead of just considering that if not provided it has no effect?

Plus it will stay accurate if at some point we extend the impact of this property to continuous axes